doc/rados/operations/poolsets.rst

```diff
@@ -0,0 +1,73 @@
+
+======================================
+Automated PG management using poolsets
+======================================
+
+What is a poolset?
+==================
+
+Poolsets are simply groups of pools, used for a shared purpose (such as a
+CephFS filesystem, or an RGW Zone).  They simplify administration by
+enabling you to create your pools in a single command, and they enable
+the Ceph to do a better job of configuring placement groups, by using
```

Comment:
"they enable the Ceph" sounds weird. Maybe "they enable Ceph" or "they enable the Ceph cluster"?